roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -152,10 +149,6 @@ kiali_defaults:
       cache_duration: 7
       cache_enabled: true
       cache_expiration: 300
-      component_status:
-        app_label: "prometheus"
-        is_core: true
-        namespace: ""
```

Comment:
Does Prometheus have a "core_component" field also? If so, it should be specified here and documented in the `kiali_cr.yaml` so users know about it.